Foundation/NSPathUtilities.swift

```diff
@@ -30,7 +30,11 @@ public func NSTemporaryDirectory() -> String {
             return tmpdir
         }
     }
+    #if os(Android)
```

Comment:
The following would also need to be changed
	modified:   Foundation/Data.swift
	modified:   Foundation/DateFormatter.swift
	modified:   Foundation/NSData.swift
	modified:   Foundation/NSObjCRuntime.swift
	modified:   Foundation/NSPathUtilities.swift
	modified:   Foundation/NSUserDefaults.swift
	modified:   Foundation/NumberFormatter.swift
	modified:   Foundation/Process.swift